test/base/FakeIdentifierActionFailed.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package base
+
+import controllers.actions.IdentifierAction
+import models.requests.IdentifiedRequest
+import play.api.mvc._
+import uk.gov.hmrc.auth.core.InsufficientEnrolments
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class FakeIdentifierActionFailed @Inject() (bodyParsers: PlayBodyParsers) extends IdentifierAction {
```

Comment:
do we need this? 🤔

Reply:
but controllers shouldnt be touched if auth fails, so what is actually being tested?